packages/markdown-it-synapse-table/package.json

```diff
@@ -38,9 +37,11 @@
     "markdown-it": "^14.1.0",
     "markdown-it-testgen": "~0.1.6",
     "tslib": "^2.6.2",
-    "tsup": "^8.0.2",
```

Comment:
Prefer Vite for build over tsup (Vite/Rollup handles mixed CJS/ESM much better than tsup/esbuild)